src/storage/DatedIrsVamm.sol

```diff
@@ -369,6 +375,10 @@ library DatedIrsVamm {
     { 
         (LPPosition.Data storage position, bool newlyCreated) = LPPosition._ensurePositionOpened(accountId, tickLower, tickUpper);
         if (newlyCreated) {
+            uint256 positionsPerAccountLimit = PoolConfiguration.load().positionsPerAccountLimit;
+            if (self.vars.positionsInAccount[accountId].length >= positionsPerAccountLimit) {
```

Comment:
it feels a bit more intuitive to have this check after the `push` operation and to convert it to a strict comparison.

Reply:
I prefer doing the check before changing state, let's keep it like this